shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Serialisation/Tripping/CBOR.hs

```diff
@@ -197,11 +236,12 @@ tests =
         prop_roundtrip_LEDGER_PredicateFails,
       testProperty "roundtrip Protocol State" prop_roundtrip_PrtclState,
       testProperty "roundtrip Ledger State" prop_roundtrip_LedgerState,
-      testProperty "roundtrip NewEpoch State" prop_roundtrip_NewEpochState,
+      -- testProperty "roundtrip NewEpoch State" prop_roundtrip_NewEpochState,
```

Comment:
Is there a reason this is still commented?

Reply:
I  uncommented, it passed, so a squashed everything into one commit.